src/ParseConfig.js

```diff
@@ -95,10 +95,10 @@ class ParseConfig {
    * @return {Promise} A promise that is resolved with a newly-created
    *     configuration object or with the current with the update.
    */
-  static save(attrs: { [key: string]: any }) {
+  static save(attrs: { [key: string]: any }, masterKeyOnly: { [key: string]: any }) {
```

Comment:
I realized that attrs and masterKeyOnly aren't documented in the comment above. Can you add it and run `npm run docs` and check if they are generated?

Reply:
added docs, generated successfully